common/app/views/fragments/guBand.scala.html

```diff
@@ -0,0 +1,23 @@
+<div class="gu-band">
+    <div class="gs-container gu-band__inner">
+        <div class="gu-meta">
+            <span class="gu-meta__label">Paid Content</span>
+            <div class="gu-comlabel gu-meta__more has-popup">
+                <button class="u-button-reset gu-comlabel__btn popup__toggle" data-toggle="gu-compopup--test1">
+                    About
+                    <svg width="12" height="9" viewBox="-0.525 -4 24 18"><path d="M23.2.7L12.7 9.1l-1.1.9-1.1-.898L0 .7.5 0l11.1 6.3L22.7 0l.5.7z"/></svg>
+                </button>
+                <div class="popup is-off gu-compopup gu-compopup--test1 gu-stylebox">
+                    <h3 class="gu-compopup__title">Paid stories are paid for and controlled by an advertiser</h3>
+                    <a class="gu-compopup__link" href="#">
+                        Learn more about the Guardian’s funding from outside parties
+                        <svg width="30" height="30" viewBox="0 0 30 30"><path d="M22.8 14.6l-7.6-7.6-.7.7 5.5 6.6h-14v1.5h14l-5.5 6.6.7.7 7.6-7.6v-.9"></path></svg>
```

Comment:
Those svgs should be generated as described here https://github.com/guardian/frontend/blob/master/docs/inline-svgs.md (`JavaScript templates` section) later on when you will start using templates and JS.